### ISSUE TITLE
Failed to start console-setup.service - Set console font and keymap.

### ISSUE THREAD

**Issue by Acheron17** [linuxmint/lmde6-beta#16]
Complete error message is:
Sep 14 21:51:43 backstop systemd[1]: console-setup.service: Main process exited, code=exited, status=1/FAILURE
Sep 14 21:51:43 backstop systemd[1]: console-setup.service: Failed with result 'exit-code'.
Sep 14 21:51:43 backstop systemd[1]: Failed to start console-setup.service - Set console font and keymap.

This is seen on the USB live boot as well as a freshly installed system.  Consequence is that the terminal font does not look as nice and full as per all previous versions of LM Cinnamon/LMDE.  LM has always had the best looking terminal out of the box.

**Comment by clefebvre** [comment]
Thanks @Harry-W-Haines-III, I had a feeling the terminal font looked different alright. OK we'll look into that.

**Comment by clefebvre** [comment]
and now I can't reproduce it anymore.. typical.

**Comment by clefebvre** [comment]
It looks like the change of font was done on purpose in Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1028643

**Comment by clefebvre** [comment]
Can you get back to good looking fonts by removing fonts-noto-mono? (you'll need a logout).

**Comment by clefebvre** [comment]
Ok, it looks like Debian's fixed this in testing/unstable but it's not backported in stable yet.

https://tracker.debian.org/pkg/fontconfig

```
fontconfig (2.14.2-5) unstable; urgency=medium

  * Team upload
  * Clean up fc*.sgml files generated during build (closes: #1044293)
  * Drop claim that DejaVu is Debian default (closes: #1042513)
  * Make configure files reflect upstream filename change
    (closes: #1050660)

 -- Gunnar Hjalmarsson <gunnarhj@debian.org>  Sun, 03 Sep 2023 18:49:22 +0200

fontconfig (2.14.2-4) unstable; urgency=medium

  * Team upload
  * Change the default monospcace font for latin scipts
    - Upstream‘s default monospace font for latin scripts is currently
      Noto Sans Mono, and that has been reflected in Debian since
      fontconfig 2.14.1-3. Due to some deficiencies in that font, we
      change it to the well established DejaVu Sans Mono. To cover
      situations when the fonts-dejavu-mono package is not installed,
      we put the Noto Mono font — included in the fonts-noto-mono
      package — as the second one in the list. (Closes: #1028897)

 -- Gunnar Hjalmarsson <gunnarhj@debian.org>  Sun, 20 Aug 2023 13:37:33 +0200
```

**Comment by clefebvre** [comment]
We could.. 

- override the fontconfig choice in /etc/fonts/conf.d/... I'm not really keen on that.
- backport the fontconfig patches from Debian, but they're likely to land soon or later.

or much simpler, instead of telling Cinnamon to use the default mono font, we could simply set it to use DejaVu Sans Mono 10. This seems like the best approach to me.

**Comment by Acheron17** [comment]
For what it is worth, this issue also affects the way the fixed fonts look in xed in a live USB session.  Thank you for investigating.

**Comment by clefebvre** [comment]
That's ok. Xed defaults to the system font, which is set by Cinnamon.

**Comment by clefebvre** [comment]
Fixed in mint-artwork 1.7.6.
